src/ansys/dpf/composites/data_sources.py

```diff
@@ -149,14 +149,14 @@ def _get_rst_list(value: Union[list[_PATH], _PATH]) -> list[_PATH]:
         return value  # type: ignore
 
 
-# roosre June 2023: todo add deprecation warning where composite definition label is used
 @dataclass(frozen=True)
 class CompositeDataSources:
     """Provides data sources related to the composite lay-up.
 
     Parameters
     ----------
     rst:
+        Result file. Currently only RST (MAPDL) is supported.
 
     material_support:
         NOTE: The 'material_support' is explicitly listed since it is currently not
```

Comment:
```suggestion
        NOTE: The ``material_support`` parameter is explicitly listed because it is currently not
```
I would omit "NOTE:" if this is a parameter description.